examples/labs/orientation/orientation.h

```diff
@@ -27,10 +26,11 @@
  * POSSIBILITY OF SUCH DAMAGE.
  **/
 
+#ifndef ORIENTATION_H
+#define ORIENTATION_H
```

Comment:
While we are at it, might as well change it to `#pragma once` at the top (same for `orientationview.h`).